exp-player/addon/components/exp-physics-intro/template.hbs

```diff
@@ -30,12 +28,6 @@
             select a privacy level for your webcam recording.
         </p>
         <br>
-        <p style="color:blue;">
```

Comment:
With the removal of the text, is it safe to also clean up the other sections dealing with confirmation code?

https://github.com/CenterForOpenScience/exp-addons/pull/141/commits/6bcc13378ab3b475a3234b56a02478fca875d625

(We try to remove dead code before we forget about it)